README.md

```diff
@@ -1,18 +1,692 @@
+    {
+      "type": "h2", // can go as deep as h4
+      "id": "clusters/how-to-install-temporal-cli"
+    },
+    {
+      "type": "h2",
+      "id": "go/add-sdk"
+    },
+    {
+      "type": "p",
+      "id": "go/how-to-use-the-go-sdk"
+    },
+    {
+      "type": "h3",
+      "id": "go/api-reference-go"
+    },
+    {
+      "type": "h3",
+      "id": "go/code-samples"
+    },
+}
+```
+
+### How to use DACX
+
+**What is DACX?**
+
+DACX is short for “Docs as code”.
+All the logic of the DACX tool is built into the Assembly Workflow.
+
+**Why use DACX?**
+
+DACX is similar to [Snipsync](https://github.com/temporalio/snipsync), except that DACX lets you write the documentation within the code.
+Rather than curating the information and code within the documentation repository, you curate the information within the source code's repository.
+The Assembly Workflow then generates Markdown guides from the narrative inlined with the code.
+As a result, DACX provides rich content in multiple places.
+
+To start using DACX, choose a source repo.
+
+- Go → https://github.com/temporalio/documentation-samples-go
+- Python → https://github.com/temporalio/documentation-samples-python
+- Java → TODO
+- TypeScript → TODO
+
+You can also register a new source repo with the Assembly Workflow.
+
+Add your repository under the `documentation_samples_repos` section of the [Assembly configuration](https://github.com/temporalio/documentation/blob/main/assembly/config.json) file.
+
+Specify a branch name in the `ref` field if your changes are not merged into the main branch of the source repo.
+
+Example:
+
+```go
+{
+  "org": "your-org" // for us that temporalio
+  "name": "your-repo" // name of the repository
+  "ref" : "your-branch-name" // name of your branch - optional
+  // if a branch is not specified, main is used
+}
+```
+
+#### Identify DACX files
+
+The Assembly Workflow identifies DACX files by the inclusion of `_dacx` in filenames.
+In your source repository, add `_dacx` at the end of the file name, but before the file extension, to identify files using DACX.
+
+For example, `my_main_workflow_file_dacx.go`.
+
+This must be done for every file that contains DACX.
+
+#### Write documentation using vanilla Markdown
+
+In the source repository, write your documentation as Markdown using multiline comments.
+
+The Assembly Workflow identifies all multiline comments as Markdown documentation.
+
+Single-line comments are treated the same as the rest of the code.
+
+Single line commenting is still highly encouraged to make the code understandable.
+
+For example:
+
+```go
+/*
+This is treated as Markdown documentation.
+
+Format this information as you would like any other Markdown content.
+
+Since Markdown maintains readablibilty, anyone can read this, even when a [link](/somelink) is present.
+*/
+
+// MyCode is my code and this is treated the same as the code
+func MyCode () error {
+ // ...
+}
+```
+
+#### Use `@dacx` comments to identify info nodes
```

Comment:
```suggestion
#### Use `@dacx` comments to identify information nodes
```
However, the short form "info nodes" appears fairly regularly from this point forward, so I'm conflicted about whether to expand "info" into "information" throughout, equivalence "information node" and "info node" at the first occurrence, use "info node" throughout (which I kinda like), or simply stet.